e2e/user_managed_deps/README.md

```diff
@@ -0,0 +1,53 @@
+# User managed dependencies
+
+This example shows how to write a simple Node.js program using nodejs rules from source and user managed npm dependencies.
```

Comment:
maybe this one should be under examples/ ?

Reply:
👍 moved. plus e2e is getting slow on CI so that's better. should shard the e2e tests in CI in follow up PR.